aten/src/ATen/native/cuda/linalg/CUDASolver.h

```diff
@@ -9,13 +9,6 @@
 
 #if defined(CUDART_VERSION) || defined(ROCM_VERSION) && ROCM_VERSION >= 50200
 
-#ifdef CUDART_VERSION
-#define CONST_QUALIFIER const
-#else
-// hipSOLVER functions in current ROCM (5.2) miss CONST_QUALIFIER qualifiers
-#define CONST_QUALIFIER
```

Comment:
@alugorey Actually, this PR shouldn't have been merged, because it breaks PyTorch build on ROCm5.2: http://rocmhead.amd.com:8080/job/pytorch/job/pytorch-build/283/consoleText
```
Aug 20 06:53:23 /var/lib/jenkins/pytorch/aten/src/ATen/native/hip/linalg/HIPSolver.cpp:789:65: error: invalid conversion from ‘const float*’ to ‘float*’ [-fpermissive]
Aug 20 06:53:23   789 |   TORCH_CUSOLVER_CHECK(hipsolverDnSpotrs(handle, uplo, n, nrhs, A, lda, B, ldb, devInfo));
Aug 20 06:53:23       |                                                                 ^
Aug 20 06:53:23       |                                                                 |
Aug 20 06:53:23       |                                                                 const float*
```